examples/proxy/record-validation/load-balancer/README.md

```diff
@@ -0,0 +1,84 @@
+# Streams for Apache Kafka Proxy Record Validation, exposed by External Load Balancer
```

Comment:
Whats your thinking behind producing the the same record validation example but with a different way of exposing the proxy? 

To me how the proxy is exposed is orthogonal at best to how record validation is configured/works.

Reply:
Hmm your leveraging the RHEL distribution in this example to run the tooling locally.

Reply:
Yes, admittedly weird. Can you think of a better way given the state of Streams right now?

Reply:
> Whats your thinking behind producing the the same record validation example but with a different way of exposing the proxy? 
> 
> 
> 
> To me how the proxy is exposed is orthogonal at best to how record validation is configured/works. 

My motivation was to let a customer with off cluster or on cluster apps get going quickly

Like I said on the other PR,  we need multiple listeners.

Reply:
I think your motivation is reasonable and I struggle to think of another way to achieve it. I think maybe rename cluster-ip to on-cluster and load balancer to off cluster so that distinction is obvious to users?

I still think they are orthogonal to the filter examples.

Reply:
Raises as #513